src/test/java/com/hocs/test/pages/login/LoginPage.java

```diff
@@ -15,6 +15,9 @@
     @FindBy(id = "password")
     private WebElementFacade passwordField;
 
+    @FindBy(xpath = "//div[@class='error-summary']//ul//li[text()='Invalid username or password.']")
```

Comment:
I think this is updated in a later commit but if not can probably start this at `li[text()=...`